modules/endorspy/meta.yml

```diff
@@ -0,0 +1,54 @@
+name: "endorspy"
+description: endorS.py calculates endogenous DNA from samtools flagstat files and print to screen
+keywords:
+  - endogenous DNA
+  - ancient DNA
+tools:
+  - "endorspy":
+      description: "endorS.py calculates endogenous DNA from samtools flagstat files and print to screen"
+      homepage: "https://github.com/aidaanva/endorS.py"
+      documentation: "https://github.com/aidaanva/endorS.py"
+      tool_dev_url: "https://github.com/aidaanva/endorS.py"
+      doi: ""
+      licence: "['GPL v3']"
+
+input:
+  # Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+
+  - stats:
+      type: file
+      description: output of samtools flagstat in a txt file (at least one required).
+      pattern: "*.{flagstat}"
+
+  - stats_optional:
+      type: file
+      description: output of samtools flagstat in a txt file. If two files are supplied,
+        the mapped reads of the second file is divided by the total reads in the first, since it
+        assumes that the <samplefile.stats> are related to the same sample. Useful after BAM filtering
```

Comment:
```suggestion
      description: output of a second samtools flagstat run in a text file. If two files are supplied,
        the mapped reads of the second file is divided by the total reads in the first, since it
        assumes that the <samplefile.stats> are related to the same sample. Useful after BAM filtering.
```